src/ostorlab/testing/agent.py

```diff
@@ -205,8 +209,17 @@ def agent_run_mock(mocker) -> AgentRunInstance:
 
     def mq_send_message(key, message):
         # we need to remove the last part of the key f'{selector}.{uuid.uuid1()}'
```

Comment:
docstring

Reply:
This is overriding an existing documented method.